### PR TITLE
Features

### PR DESCRIPTION
**Pull request by edwinmoradian90**
This is the entire project. I redid it since I switched partners and I guess I never turned it in. 
The project contains the clone of a youtube page, including all the specifics.

### CODE REVIEW

**Comment by edwinmoradian90** [comment]
hey, @Oghenebrume50 thanks!. Yes, I switched partners near the very beginning of the course. When I was given a new partner, he was already four weeks ahead of me, so I started with him. Which is why I have to do the first for solo.

**Comment by edwinmoradian90** [comment]
So I have stickler for my other projects, and it gives a lot of trivial errors, such as spaces after colons etc. What do you recommend as far as settings go? On my other project is gave 859 mostly trivial errors, how would we fix something like that without wasting an enormous amount of time, or am I missing something? thanks

**Comment by samgaco** [comment]
reviewing

File: index.html [hunk 1]
```diff
@@ -1,12 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Embedding Videos</title>
+  <meta charset="UTF-8" />
```
Comment (samgaco) [inline index.html]: It seems like you had problems with indentation 

> Many IDE can help you with this task.
> This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):
> 
> Screenshot of how to format code using VS Code

**Comment by edwinmoradian90** [comment]
Cool thanks, yeah idk I use prettier already, :/

**Pull request merged**